### ISSUE TITLE
_CastError (Null check operator used on a null value)

### ISSUE THREAD

**Issue by kevinxxq** [leanflutter/window_manager#300]
Hi There,
The issue can be reproduced as below:
1. turn on "catch all exceptions";
2. use the example in the package and replace the path with published version: 
```
  # window_manager:
  #   # When depending on this package from a real application you should use:
  #   #   window_manager: ^x.y.z
  #   # See https://dart.dev/tools/pub/dependencies#version-constraints
  #   # The example app is bundled with the plugin so we use a path dependency on
  #   # the parent directory to use the current plugin's version.
  #   path: ../
  window_manager: ^0.3.0
```

3. debug. the error will be caught.

![error_1_window_manager](https://user-images.githubusercontent.com/55800697/218184533-4d5f6a05-f527-469b-a5c6-e6ed4130dd7d.png)

The eventname is "show"

Flutter Doctor:
[✓] Flutter (Channel stable, 3.7.3, on Microsoft Windows [版本 10.0.22621.1194], locale zh-CN)
[✗] Windows Version (Unable to confirm if installed Windows version is 10 or greater)
[✓] Android toolchain - develop for Android devices (Android SDK version 32.1.0-rc1)
[✓] Chrome - develop for the web
[✓] Visual Studio - develop for Windows (Visual Studio Community 2022 17.4.0)
[✓] Android Studio (version 2022.1)
[✓] VS Code (version 1.73.1)
[✓] Connected device (3 available)
[✓] HTTP Host Availability

After changing it to 0.2.9, the error will disappear.

Thank you.

**Comment by lijy91** [comment]
Fixed in 0.3.1, please update

**Comment by kevinxxq** [comment]
Thank you!
